src/databricks/labs/remorph/transpiler/execute.py

```diff
@@ -209,17 +209,17 @@ async def _do_transpile(
 
 
 def verify_workspace_client(workspace_client: WorkspaceClient) -> WorkspaceClient:
-    # pylint: disable=protected-access
     """
     [Private] Verifies and updates the workspace client configuration.
 
     TODO: In future refactor this function so it can be used for reconcile module without cross access.
     """
-    product_info = workspace_client.config._product_info
+
+    # Using reflection to set right value for _product_info for telemetry
+    product_info = getattr(workspace_client.config, '_product_info')
     if product_info[0] != "remorph":
-        product_info[0] = "remorph"
-    if product_info[1] != __version__:
-        product_info[1] = __version__
```

Comment:
It's not the point of this PR, given that we're replacing it, but I'm mildly surprised that this worked… I think it's a tuple, in which case assigning to the elements individually won't work.

Reply:
it probably didn't, but that code is never called anyway... this was to fix formatting alerts